packages/runner/lib/exec.unit.test.ts

```diff
@@ -126,14 +131,15 @@ describe('Exec', () => {
         expect(res.success).toEqual(false);
     });
 
-    it('should return a formatted error when receiving an AxiosError (without a body)', async () => {
+    it('should return a script_network_error when receiving an AxiosError (without a body)', async () => {
         const nangoProps = getNangoProps();
         const jsCode = `
         fn = async (nango) => {
-            await nango.get({
-                endpoint: '/',
-                baseUrl: 'https://example.dev/'
-            })
+            const err = new Error("Something broke");
+            err.isAxiosError = true;
+            err.code = "ECONNREFUSED";
+
+            throw err;
```

Comment:
I'm not sure why would you fake this test that was working before?

Reply:
It's a unit test. Unit tests aren't supposed to make network calls. Also relying on a network failure to a domain that could appear is kind of asking for a flaky test.

Reply:
Yeah, I have my doubts on us relying on that result anyway. From one side ECONNREFUSED, etc. come out of the node http stack pretty consistently and a lot will have changed in the ecosystem if that output changes, but from the other side I'm not sure we want to rely on that output from the API anyway, since it could change. At this point though we are relying on whether a response was received or not, which seems to be the Axios recommended method for knowing if it was an actual response with an error status or if it was a network error.